Funcky.Test/Extensions/EnumerableExtensions/PartitionTest.cs

```diff
@@ -0,0 +1,27 @@
+using System.Linq;
+using Funcky.Extensions;
+using Xunit;
+
+namespace Funcky.Test.Extensions.EnumerableExtensions
+{
+    public sealed class PartitionTest
+    {
+        [Fact]
+        public void ReturnsTwoEmptyEnumerablesWhenSourceIsEmpty()
+        {
+            var (evens, odds) = Enumerable.Empty<int>().Partition(IsEven);
+            Assert.Empty(evens);
+            Assert.Empty(odds);
+        }
+
+        [Fact]
+        public void PartitionsItemsIntoTruesAndFalses()
+        {
+            var (evens, odds) = Enumerable.Range(0, 6).Partition(IsEven);
+            Assert.Equal(new[] { 0, 2, 4 }, evens);
+            Assert.Equal(new[] { 1, 3, 5 }, odds);
+        }
```

Comment:
Maybe add a test case with Functional.True or False, to generate a case where one partition is Empty

Reply:
Addressed in 16772010e459a06330263071e2d8c0cea4602798